tool/gravity/cli/user.go

```diff
@@ -163,7 +163,7 @@ func inviteUser(env *localenv.LocalEnvironment, username string, roles []string,
 		return trace.Wrap(err)
 	}
 
-	fmt.Printf("Signup token has been created and is valid for %v hours. Share this URL with the user:\n%v\n\nNOTE: make sure this URL is accessible!\n",
+	fmt.Printf("Signup token has been created and is valid for %v  Share this URL with the user:\n%v\n\nNOTE: make sure this URL is accessible!\n",
```

Comment:
I think you didn't mean to remove the dot.

Reply:
If I left the dot, it would say "8h0m0s ." , right? That doesn't look quite right either, due to the extra space. Would concatenating the strings like this work in golang?

`... valid for %v" + ". Share this URL ...`

Reply:
No, just leave the dot after `%v`.

```go
fmt.Printf("Signup token has been created and is valid for %v. Share ...")
```

Reply:
Done, thanks.